app/assets/javascripts/admin/addon/components/site-setting.gjs

```diff
@@ -226,6 +264,14 @@ export default class SiteSettingComponent extends Component {
     return this.setting.staffLogFilter;
   }
 
+  get canUpdate() {
+    if (!this.status || this.status === "completed") {
```

Comment:
If I reload the page, wouldn't the component show up as not having any `status` and thus be marked as updatable?

Reply:
I guess we can ignore this for now since the background job only has a cluster concurrency of 1. There is the risk that the original job may fail though and the new job runs before the old job. However, I think we have already decided to accept the risk for now.